spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Recurrent.scala

```diff
@@ -108,9 +110,36 @@ class Recurrent[T : ClassTag]()
           c.parameters()._2(i).storage().set(params._2(i).storage())
           i += 1
         }
+
+        dropouts.append(findDropouts(c))
       }
-      c
     })
+
+    val stepLength = dropouts.length
+    for (i <- dropouts.head.indices) {
+      val head = dropouts.head.apply(i)
+      val noise = head.noise.resize(1)
```

Comment:
Because at start I made the noise share the same storage by `current.noise.set(noise)`. The size of noise is empty is empty, and `NullPointerException` was thrown, so I give the tensor a initial size. But I thought is it better to set the noises point to the same tensor by `current.noise = noise`. Now the resize is not necessary and I have removed it.